### PR TITLE
plant_gen + biome at HexPillar

### PR DESCRIPTION
**Pull request by adRichter**

### CODE REVIEW

File: base/src/gen/plant/mod.rs [hunk 1]
```diff
@@ -19,10 +20,14 @@ impl PlantGenerator {
...
+    pub fn rand<R: Rng>(rng: &mut R, biome: Biome) -> Self {
+        PlantGenerator::Tree(TreeGen::rand(rng, biome))
     }
 }
+
+// impl Rand for PlantGenerator {
+//     fn rand<R: Rng>(rng: &mut R) -> Self {
+//         PlantGenerator::Tree(TreeGen::rand(rng))
+//     }
+// }
```
Comment (jonas-schievink) [inline base/src/gen/plant/mod.rs]: This can be removed

File: client/src/renderer.rs [hunk 2]
```diff
@@ -157,7 +157,7 @@ impl Renderer {
         // 0: Disable Bloom
         // 1: Enable Bloom
         // 2: Show only Bloom Map
-        let bloom_state = 1;
+        let bloom_state = 0;
```
Comment (jonas-schievink) [inline client/src/renderer.rs]: Do you need to globally disable it? Why?

File: base/src/gen/plant/tree.rs [hunk 3]
```diff
@@ -266,20 +423,33 @@ impl TreeGen {
...
+        };
+        let preset = rng.choose(matched_preset).unwrap().clone();
 
         TreeGen {
             preset: preset,
             branches: Vec::new(),
         }
+
+
+
```
Comment (LukasKalbertodt) [inline base/src/gen/plant/tree.rs]: Random empty lines?

File: client/src/renderer.rs (hunk 2 above)
Comment (LukasKalbertodt) [inline client/src/renderer.rs]: Yep, please enable it on master :stuck_out_tongue:

**Pull request merged**